src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/CompilerGeneratedState.cs

```diff
@@ -111,58 +112,87 @@ void ProcessMethod(MethodDesc method)
                         while (reader.HasNext)
                         {
                             ILOpcode opcode = reader.ReadILOpcode();
-                            MethodDesc? lambdaOrLocalFunction;
                             switch (opcode)
                             {
                                 case ILOpcode.ldftn:
                                 case ILOpcode.ldtoken:
                                 case ILOpcode.call:
                                 case ILOpcode.callvirt:
+                                case ILOpcode.initobj:
```

Comment:
initobj is not like the others - it takes a type token, not method token. It doesn't look right given what we do below.

Reply:
I made this mistake in a previous change here and now just duplicated it. Removed initobj.